README.md

```diff
@@ -2,59 +2,86 @@ dcrwords
 
-**[Consensus rules voting](https://docs.decred.org/getting-started/user-guides/agenda-voting/)** : Changes to Decred's consensus rules can only be made through an on-chain voting process which lasts for around one month, the change is made if at least 75% of votes are in favor. 
+**[Consensus rules voting](https://docs.decred.org/getting-started/user-guides/agenda-voting/):** 
+Changes to Decred's consensus rules can only be made through an on-chain voting process which lasts for around one month. The change is made only if at least 75% of votes are in favor. 
 
 **Agenda voting:** see consensus rules voting
 
-**Quorum:** A minimum level of participation required in order for a decision-making process to produce a valid outcome. Changes to the consensus rules require at least 10% of votes to be for or against the change in order to be valid.
+**Quorum:** 
+The minimum level of participation required in order for a decision-making process to produce a valid outcome. Changes to the consensus rules require at least 10% of votes to be for or against the change in order to be valid.
 
-**Rule change proposal:** Consensus rules votes concern a specific rule change proposal - which must be implemented in latent code within the software that the network's nodes are running. If the proposal passes, the latent code activates one month later.
+**Rule change proposal:** 
+A proposal to change the consensus rules of the Decred blockchain. Rule change proposals must be implemented in latent code within the software running the network's nodes. If the proposal passes, the latent code activates one month later. (SETH: review for accuracy)
 
-**[Decred change proposal (DCP)](https://github.com/decred/dcps):**  Design document that describes potential protocol or consensus changes to Decred. DCPs primarily serve for documentation, fostering cross-implementation compatibility, and helping ensure proper engineering rigor is followed.
+**[Decred change proposal (DCP)](https://github.com/decred/dcps):**  
+A design document that describes potential protocol or consensus changes to Decred. DCPs primarily serve for documentation, fostering cross-implementation compatibility and helping ensure proper engineering rigor is followed.
 
-Stake version interval: 
+**Stake Version Interval (SVI):** 
+An interval of blocks 2016 blocks (~1 week) which is used to determine if a vote on concensus rules can begin. Before a vote on consensus rule changes can begin, 75% of tickets that vote during a Stake Version Interval (SVI) must be using software that contains the latent software change being proposed. 
```

Comment:
I was going to include the mining node requirement, but didn't because that is independent of the SVI. I believe the interval for PoW miners to upgrade is 1,000 blocks. I do think it's relevant here, but it's a tradeoff between keeping the definition concise and "pure", and not misleading the reader by implying that rule changes can begin if 75% of tickets vote for it (without consideration by PoW miners). I tried to be careful in wording not to imply that, but can see how it might be implied anyway. Open to either course.